# Single viewpoint for observing a frontier cluster
geometry_msgs/Point position
float64 yaw
int32 coverage
float64 distance_to_centroid